x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexing/IndexerJobStats.java

```diff
@@ -84,6 +84,11 @@ public long getOutputDocuments() {
         return numOuputDocuments;
     }
 
+    @Deprecated
+    public long getNumRollups() {
```

Comment:
I'm not entirely positive if this is required... but I think it is.  We expose the IndexerJobStats object in the GetRollupJob API, so I think we need to keep this method around because anyone using the Java client could potentially be using it.

I think :)